docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -1422,7 +1422,71 @@ public GlobalFilter customGlobalPostFilter() {
 
 === Writing Custom Route Locators and Writers
 
-TODO: document writing Custom Route Locators and Writers
+If you create a custom ``RouteDefinition``, you can use ``RouteDefinitionWriter`` to add and delete Route.
```

Comment:
I think this should be single back-tick.

Reply:
Sorry about that, I was misunderstanding this. I will rwirte that. :flushed: